python_modules/dagster/dagster/core/storage/event_log/sql_event_log.py

```diff
@@ -634,13 +634,9 @@ def get_event_records(
             query = query.limit(limit)
 
         if ascending:
-            query = query.order_by(
-                SqlEventLogStorageTable.c.timestamp.asc(), SqlEventLogStorageTable.c.id.asc()
-            )
+            query = query.order_by(SqlEventLogStorageTable.c.id.asc())
```

Comment:
did you verify that this does a reasonable plan? seems very plausible to me that it would

Reply:
i have not run explain on this

Reply:
I linked some explains from our DB in the internal version of this PR